src/quad_gl.rs

```diff
@@ -995,32 +993,46 @@ pub struct Texture2D {
     texture: miniquad::Texture,
 }
 
+/// # Constructors
```

Comment:
We never used # Constructors / # Methods separation before and I am not really sure about that separation

Reply:
Sure, we can leave that out